diesel_cli/src/main.rs

```diff
@@ -405,6 +405,14 @@ fn run_infer_schema(matches: &ArgMatches) -> Result<(), Box<dyn Error>> {
         config.with_docs = true;
     }
 
+    if let Some(sorting) = matches.value_of("column-sorting") {
+        match sorting {
```

Comment:
We probably want to use claps buildin enum support here, instead of doing a manual matching?

Reply:
How would that work? `possible_values` [only takes strings](https://docs.rs/clap/2.33.0/clap/struct.Arg.html#method.possible_values). For other commands that currently use `possible_values` (like MIGRATION_FORMAT) we currently also match on the string.

Clap 3 will have `structopt` integrated, that could become handy for less stringly typed argument parsing. But it's not released yet.

Reply:
https://docs.rs/clap/2.33.0/clap/macro.arg_enum.html